### DISCUSSION TITLE
Looking for githubstatus Object?

### DISCUSSION THREAD

**Discussion by yoyomeng2** [PyGithub/PyGithub#2023] [Q&A] [answered]
Looking for something that wraps githubstatus.com like https://www.githubstatus.com/api/v2/summary.json

Still digging but haven't found it yet.

Thanks!

**Comment by xmo-odoo** [comment, marked as answer]
I don't think you're likely to find it: pygithub specifically says that it exposes [the github REST ("v3") API](https://docs.github.com/en/rest).

githubstatus seems to have its own completely separate (and even less documented or specified) API at https://www.githubstatus.com/api
